CMakeLists.txt

```diff
@@ -343,6 +343,12 @@ if (NOT LLVM_ENABLE_RTTI)
   endif()
 endif()
 
+find_package(SQLite3)
```

Comment:
This should be a little further down and have a dedicated section with header similar to libcap/libutil/....